shared-bindings/vectorio/Polygon.c

```diff
@@ -16,15 +17,21 @@
 
 
 //| class Polygon:
```

Comment:
I think you might want to add `class Polygon(VectorShape):` so that the class inherits the shared attributes.

Reply:
Hmm that’s an interesting thought; does it not pull the docs from the properties? They are attached to the concrete shapes directly, not as base class properties. Are you thinking about dir() or something else?

Reply:
The API docs aren't that smart. I think `dir()` will work like you want. These docs aren't: https://circuitpython.readthedocs.io/en/latest/shared-bindings/vectorio/index.html

They are solely based on the python stub definitions that are prefixed with `//|` in the C files.